UnbxdSearch_IO.js

```diff
@@ -1430,10 +1430,7 @@ var unbxdSearchInit = function (jQuery, Handlebars) {
             this.params['extra']['version'] = 'V2';
 
             if (this.options.facetMultilevel) {
-                this.params['extra']['facet.multilevel'] = 'categoryPath';
-                this.params['extra']['f.categoryPath.displayName'] = this.options.facetMultilevelName;
-                this.params['extra']['f.categoryPath.max.depth'] = this.options.categoryDepth;
-                this.params['extra']['f.categoryPath.facet.limit'] = this.options.categoryLimit;
+                /** With self serve user can configure category facets in the portal itself */
                 this.params['extra']['f.categoryPath.facet.version'] = 'V2';
```

Comment:
missed this..

Reply:
@anantheshadiga As discussed, Removal of this gives v1 response which new sdk is not tested for. Also, if backend enables v2 for other facets i.e. text and range, then sdk will break as it does not handle v2 for the same.

Reply:
Removal gives v2 response now.